tests/unit/test_charm.py

```diff
@@ -238,63 +278,33 @@ def test__loki_config_non_empty(self):
             ]
         }
 
-        self.assertDictEqual(self.harness.charm._loki_config(), expected)
-
-    def test__config_file_without_loki(self):
-        self.maxDiff = None
-        expected = {
-            "integrations": {
-                "agent": {
-                    "enabled": True,
-                    "relabel_configs": REWRITE_CONFIGS,
-                },
-                "prometheus_remote_write": [],
-            },
-            "prometheus": {
-                "configs": [{"name": "agent_scraper", "remote_write": [], "scrape_configs": []}]
-            },
-            "server": {"log_level": "info"},
-        }
-        self.assertDictEqual(yaml.safe_load(self.harness.charm._config_file()), expected)
+        self.assertDictEqual(yaml.safe_load(content)["loki"], expected)
+        self.assertEqual(path, "/etc/agent/agent.yaml")
+        self.assertEqual(self.harness.model.unit.status, ActiveStatus())
 
-    def test__config_file_with_loki(self):
+    @responses.activate
+    @patch.object(Container, "pull", new=pull_empty_fake_file)
+    @patch.object(Container, "push")
+    def test__on_logging_relation_departed(self, mock_push: MagicMock):
```

Comment:
Could you name/document the test to be more immediately obvious what it is testing? Applies to some of the other tests as well.

Reply:
Sure, I always prefer little tests anyway